spec/support/features/document_capture_step_helper.rb

```diff
@@ -3,7 +3,18 @@ def submit_images
     click_on 'Submit'
 
     # Wait for the the loading interstitial to disappear before continuing
-    expect(page).not_to have_content(t('doc_auth.headings.interstitial'), wait: 10)
+
+    begin
+      expect(page).not_to have_content(t('doc_auth.headings.interstitial'), wait: 10)
+    rescue Selenium::WebDriver::Error::StaleElementReferenceError
```

Comment:
seeing all this repeated, I feel like we could probably build a helper like

```
wait_for_content_to_disappear do |page|
    expect(page).not_to have_content(t('doc_auth.headings.interstitial'), wait: 10)
end
```
and then that helper method has these `rescue` blocks built in

Reply:
Good call, added in https://github.com/18F/identity-idp/pull/10627/commits/d8d45bf42fc3245ea49f5ab6ea5053aef534f220